### ISSUE TITLE
DOC: MIRR documentation is lacking

### ISSUE THREAD

**Issue by Kai-Striega** [numpy/numpy-financial#76]
### Issue with current documentation:

The documentation for MIRR is very limited (see below screenshot)

![image](https://github.com/numpy/numpy-financial/assets/30074037/c97b1af6-dfac-4e23-9717-2cca04a27428)

### Idea or request for content:

The documentation for MIRR should explain _what_ it is, _how_ to use it and _when_ to use it. In addition, in should supply some examples of how it should be used

**Comment by RVitalii** [comment]
**Hello,** 

Is it enough to change the description inside the docstring of the MIRR function? 

I know NumPy web docs are autogenerated , although the text inside the function differs a bit from the text on the webpage.

**Regards, 
VR**

**Comment by Kai-Striega** [comment]
Yes, changing the description inside the doc string should be good, saying that I quite like what you have done with your PR

**Comment by Kai-Striega** [comment]
Closed in #86. So closing this issue
